app/views/users/edit.haml

```diff
@@ -16,6 +13,11 @@
       %td{class: cycle('even', 'odd')}
         = f.check_box :email_notifications, class: 'width-15 vmid'
         = f.label :email_notifications, "Email about new tags"
+    - unless current_user.avatar.present?
```

Comment:
for informative-style reasons this is generally better done with a check on avatar_id so you don't load the object, but because the layout already loads the object I do not care in this specific case.